extensions/gitlab/package.json

```diff
@@ -263,6 +263,14 @@
       "placeholder": "/my/cert.pem",
       "required": false,
       "description": "Set Custom Certificate"
+    },
+    {
+      "name": "excludeTodoAuthorUsernames",
+      "type": "textfield",
+      "required": false,
+      "title": "Exclude Todo Authors",
+      "description": "Hide Todos Created By Someone",
+      "placeholder": "Username separated by comma (e.g., bot1,bot2)"
```

Comment:
I would reduce the placeholder to `e.g. bot1,bot2` because it get truncated in the UI.